fpd/fpd_bmc_bios.cc

```diff
@@ -53,6 +53,36 @@ get_shell_cmd_output(const char* cmd) {
     return result;
 }
 
+void
+Fpd_bmc_bios::select_mux() const
+{
+
+    /* Select MUX for BMC access  */
+    std::vector<std::string> helper;
+    std::vector<std::string> arg_list;
+    std::string helper_script = fpd_t::get_fpga_offset("select_mux_script");
```

Comment:
What is the reason for invoking a script versus performing the i2c transactions directly?

Reply:
These scripts were created to follow a similar model as how we used to call the helper script for mtd_upgrade for consistency. The eventual goal is to convert all of them to C/C++ code for more a direct invocation and inline with Meta's FPD requirements.

Reply:
Have created an internal JIRA to track this:
MIGSOFTWAR-4735